app/views/spree/reviews/create.js.erb

```diff
@@ -0,0 +1,13 @@
+<% if @review.errors.blank? %>
+	$("#notice-msg").html('<%= Spree.t('pages.product_detail.review.submit_successfuly')%>');
```

Comment:
The translation is not related to the product details page. I only belongs to reviews
Change this Spree.t('pages.product_detail.review.submit_successfuly')
to this
Spree.t('pages.review.submit_successfuly')